judge/views/register.py

```diff
@@ -34,7 +35,10 @@ class CustomRegistrationForm(RegistrationForm):
         newsletter = forms.BooleanField(label=_('Subscribe to newsletter?'), initial=True, required=False)
 
     if ReCaptchaField is not None:
-        captcha = ReCaptchaField(widget=ReCaptchaWidget())
```

Comment:
Rather than having this and then needing to have duplicated blocks in the template, could we keep calling this `captcha`, and only do the hcaptcha thing in an else branch?

Reply:
Sorry for letting this go mouldy, I'll get this done soon.